src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs

```diff
@@ -11,6 +11,12 @@ public class DefaultCompositeRule : IPackageVerifierRule
     {
         IPackageVerifierRule[] _rules = new IPackageVerifierRule[]
         {
+            new AssemblyHasCompanyAttributeRule(),
+            new AssemblyHasCopyrightAttributeRule(),
+            new AssemblyHasDefaultAliasAttributeRule(),
```

Comment:
Hmm reading MSDN I don't think we need DefaultAlias. It sounds like some COM thing or something. Can probably get rid of it.